lib/krikri/harvester.rb

```diff
@@ -77,5 +84,40 @@ def run
       true
     end
 
+    ##
+    # An application-wide registry of defined Harvesters.
+    class Registry
```

Comment:
I'm in favor of `Krikri::Registry`.  Seems like `#keys` would be as relevant to the existing registry as this one, yes?